notebooks/en/structured_generation_vision_languag_models.ipynb

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
<p>Can remove the second sentence here since its pretty much the same as the one above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>